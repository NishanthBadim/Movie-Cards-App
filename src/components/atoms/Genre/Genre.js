import React from 'react';

const Genre =(props) => {
    return(
        <h6 className="card-subtitle mb-2 text-muted">
            {props.genre}
        </h6>
    );
}
export default Genre;