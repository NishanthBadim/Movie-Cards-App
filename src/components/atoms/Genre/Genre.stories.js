import React from 'react'
import {withKnobs, text } from '@storybook/addon-knobs/react'
import Genre from './Genre'

export default {
    component: Genre,
    title: "Genre",
    decorators: [withKnobs]
}

export const Default = () => {
    return (
        <Genre genre={text("text","Here goes the Genre of movie")} />
    )
}