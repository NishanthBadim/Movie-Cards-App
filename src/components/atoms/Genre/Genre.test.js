import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Genre from './Genre'

test("Match the snapshot - Genre",() => {
    const input=create(<Genre />);
    expect(input.toJSON()).toMatchSnapshot();
})