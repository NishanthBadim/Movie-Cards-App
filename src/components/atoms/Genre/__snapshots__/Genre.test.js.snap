// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Genre 1`] = `
<h6
  className="card-subtitle mb-2 text-muted"
/>
`;
